_i18n/es/resources/user-guides/monero-wallet-cli.md

```diff
@@ -1,147 +1,114 @@
 
-If you are the third party (that is, someone wants to prove to you that they sent monero
-to an address), then you can check this way:
+Si tú eres el tercero (esto es, alguien quiere probarte que enviaron Monero a una dirección), entonces puedes revisarlo de esta forma:
 
     check_tx_key TXID TXKEY ADDRESS
 
-Replace `TXID`, `TXKEY` and `ADDRESS` with the transaction ID, per-transaction key, and destination
-address which were supplied to you, respectively. monero-wallet-cli will check that transaction
-and let you know how much monero this transaction paid to the given address.
+Reemplaza `TXID`, `TXKEY` y `ADDRESS` con el ID de transacción, la llave por transacción y la dirección de destino que te fueron provistas respectivamente. monero-wallet-cli revisará esa transacción y te hará saber cuánto monero se utilizó en esta transacción a la dirección dada.
 
 
-## Getting a chance to confirm/cancel payments
+## Obtener una oportunidad para confirmar/cancelar pagos
 
-If you want to get a last chance confirmation when sending a payment:
+Si deseas obtener una confirmación de última oportunidad al enviar un pago:
 
     set always-confirm-transfers 1
 
 
-## How to find a payment to you
+## Cómo encontrar un pago
 
-If you received a payment using a particular payment ID, you can look it up:
+Si recibiste un pago utilizando un ID de pago en particular, puedes verlo con:
 
     payments PAYMENTID
 
-You can give more than one payment ID too.
+Puedes dar más de un ID de pago también.
 
-More generally, you can review incoming and outgoing payments:
+De manera más general, puedes revisar pagos de entrada y salida:
 
     show_transfers
 
-You can give an optional height to list only recent transactions, and request
-only incoming or outgoing transactions. For example,
+Puedes dar una altura opcional para listar sólo transacciones recientes, y solicitar solamente transacciones de entrada o salida. Por ejemplo,
 
     show_transfers in 650000
 
-will only incoming transfers after block 650000. You can also give a height
-range.
+sólo mostrará transacciones de entrada después del block 650000. También puedes dar un rango de altura.
 
-If you want to mine, you can do so from the wallet:
+Si quieres minar, puedes hacerlo desde tu monedero:
 
     start_mining 2
 
-This will start mining on the daemon usin two threads. Note that this is solo mining,
-and may take a while before you find a block. To stop mining:
+Esto empezará a minar en el daemon utilizando dos hilos. Ten en cuenta que esto es minado en individual, y puede tomar un tiempo en encontrar un bloque. Para detener el minado:
```

Comment:
While "threads" are "hilos", in a cpu, "threads" are more commonly known as "núcleos". Is this correct?

Reply:
thread	- subproceso
Core - núcleo

Reply:
You're right.

Reply:
What do you mean by "minado en individual"? Both solo mining and pool mining software is run in individual processes, on individual computers etc so I don't think the word "individual" means anything here.

The GUI uses "Minado en solitario" which is descriptive and also makes it clearer that it's English counterpart is "solo mining".
Cfr http://context.reverso.net/traduccion/espanol-ingles/en+solitario

Maybe add this to the terminology page?

Reply:
@ordtrogen could you add it to the terminology page? I also agree with this. I made a comment about some translations that are already being worked in the [`monero-gui repository](https://github.com/monero-project/monero-gui/blob/master/translations/monero-core_es.ts). Do you know if the `monero-wallet-cli` command lines are already being worked, is [this repository/ monero/tree/master/translation](https://github.com/monero-project/monero/tree/master/translations) the translation for CLI?

Reply:
I have created a PR for it, but there's an issue with "mining" in Spanish. I'll create an issue for it in monero-translations to explain.

Yes, that's for the CLI. But I don't think it's ever been enabled, so even if you add a translation, it will never be used.

Reply:
oh ok. Thank you.

Reply:
So, do I use _subproceso_ or _núcleo_?

Reply:
I have seen in some Intel pages that _hilos_ is used for "threads", but I'm not sure.

Reply:
Foun this, could help. [Intel](https://ark.intel.com/es/products/72164/Intel-Core-i5-3230M-Processor-3M-Cache-up-to-3_20-GHz-rPGA) uses:

 - Cantidad de núcleos 2 (Núcleos es un término de hardware que describe el número de unidades de procesamiento independientes en un componente computacional individual (matriz o chip).)
 - Cantidad de subprocesos 4 (Un hilo, o hilo de ejecución, es un término de software para la secuencia de instrucciones de orden básico que puede pasar por o procesarse en un núcleo de CPU individual.)

They say, "subproceso", like the "subprocess" made by each thread, but uses the word "hilo" while addressing what is a "thread"